library/common/extensions/filters/http/socket_selection/filter.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "envoy/http/filter.h"
+
+#include "source/common/common/logger.h"
+#include "source/extensions/filters/http/common/pass_through_filter.h"
+
+#include "library/common/extensions/filters/http/socket_selection/filter.pb.h"
+#include "library/common/types/c_types.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace SocketSelection {
+
+/**
+ * Interface for socket selection controller. Shared by filter instances.
+ */
+class SocketSelectionController {
```

Comment:
Unused?

Reply:
Ah, good catch, this is left over from an earlier approach. Will remove.